stories/core_components/BarChart.tsx

```diff
@@ -5,12 +5,22 @@ import { screenshotWrap } from '@gooddata/test-storybook';
 import { BarChart } from '../../src/index';
 import { onErrorHandler } from '../mocks';
 import { CUSTOM_COLORS } from '../data/colors';
-import { ATTRIBUTE_1, ATTRIBUTE_2, MEASURE_1, MEASURE_1_WITH_ALIAS, MEASURE_2 } from '../data/componentProps';
+import {
+    ATTRIBUTE_1,
+    ATTRIBUTE_2,
+    MEASURE_1,
+    MEASURE_1_WITH_ALIAS,
+    MEASURE_2,
+    ATTRIBUTE_1_SORT_ITEM,
+    MEASURE_2_SORT_ITEM
+} from '../data/componentProps';
+
+const dimensions = { width: 800, height: 400 };
```

Comment:
How about renaming this object to eg wrapperStyle and then passing it to each style prop istead of destructuring it?